Sofascore2025/LeagueView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Styling.swift
+//  Sofascore2025
+//
+//  Created by Ivan Dzanija on 16.03.2025..
+//
+
+import UIKit
+import SnapKit
+import SofaAcademic
+
+final class LeagueView: BaseView {
+	private let league: League
+	private let logoImageView = UIImageView()
+	private let countryLabel = UILabel()
+	private let splitImageView = UIImageView()
+	private let nameLabel = UILabel()
+	
+	init(league: League) {
+		self.league = league
+		super.init()
+	}
+	
+	override func addViews() {
+		addSubview(logoImageView)
+		addSubview(countryLabel)
+		addSubview(splitImageView)
+		addSubview(nameLabel)
+	}
+
+	override func styleViews() {
+		backgroundColor = .white
+		
+		logoImageView.contentMode = .scaleAspectFit
+		logoImageView.clipsToBounds = true
+
+		countryLabel.font = FontStyle.smallBold
```

Comment:
i onda kad stavis ekstenziju ces tu moc pisat samo ovako
```suggestion
		countryLabel.font = .smallBold
```